common/utils/src/main/resources/error/error-classes.json

```diff
@@ -1707,6 +1707,11 @@
     },
     "sqlState" : "42823"
   },
+  "INVALID_TABLE_FUNCTION_IDENTIFIER_ARGUMENT_MISSING_PARENTHESES" : {
```

Comment:
Should this be under INVALID_SQL_SYNTAX?

Reply:
Sounds good, this is done.